src/main/kotlin/org/rust/ide/statistics/RsConvertJsonToStructUsagesCollector.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.statistics
+
+import com.intellij.internal.statistic.eventLog.EventLogGroup
+import com.intellij.internal.statistic.eventLog.events.EventFields
+import com.intellij.internal.statistic.service.fus.collectors.CounterUsagesCollector
+
+@Suppress("UnstableApiUsage")
+object RsConvertJsonToStructUsagesCollector : CounterUsagesCollector() {
```

Comment:
Actually, I'd merge all code from this class to `RsCounterUsagesCollector`
If it's ok for you, don't forget to provide more specific names for log methods